<launch>

  <node pkg="sicktoolbox_wrapper" type="sicklms1" output="screen" name="sicklms1" />

  <node pkg="sickscan3d" type="motorcontrol" output="screen" name="motorcontrol" />

  <node pkg="sickscan3d" type="scan3d" output="screen"  name="scan3d">
    <remap from="scan" to="scan"/>
    <param name="tf_cache_time_secs" type="double" value="10.0" />
    <param name="max_scans" type="int" value="400" />
    <param name="ignore_laser_skew" type="bool" value="false" />
    <param name="fixed_frame" type="string" value="world" />
  </node>

</launch>

